README.md

```diff
@@ -16,21 +16,27 @@ All contributors are welcome! If you would like to contribute:
 
 ### Setup
 1. Fork this repo and clone it locally by running `git clone https://github.com/<your_username>/rideboard.git`
-2. `cd rides/`
+2. `cd csh-rideboard/`
 2. Create a python virtual environment, activate it and install requirements.
   - `virtualenv rides-venv`
   - `source rides-venv/bin/activate`
   - `pip install -r requirements.txt`
-5. You will need the _LDAP DN_, _LDAP PW_, _MySQL Databse_, and _OIDC Secret_. You can create your own if you are hosting your own application or ask me for these if you would like to contribute to CSH.
+5. You will need many credentials to run, see the exports below. You can create your own if you are hosting your own application or ask an rtps for these if you would like to contribute to CSH. I'd recommend making a config.sh to automatically do the exports you need. It is in the .gitignore so it will not go to the remote repository.
 ```
-export SERVER_NAME=127.0.0.1:8080
-export IP=127.0.0.1
-export PORT=8080
-export SQLALCHEMY_DATABASE_URI=postgresql://<LDAP DN>:<LDAP PW>@<MySQL Database>
-export OIDC_CLIENT_SECRET=<OIDC Secret>
+export SERVER_NAME=127.0.0.1:8080 # Port 5000 sometimes works better in personal development
+export IP=127.0.0.1 # Standard IP
+export PORT=8080 # Port 5000 sometimes works better in personal development
+export SQLALCHEMY_DATABASE_URI=mysql+pymysql://<LDAP DN>:<LDAP PW>@<MySQL IP:Port>/<MySQL Schema> # ESSENTIAL to run service
```

Comment:
Why mysql instead of postgres? The app is deployed against postgres, so you might want to replicate that for local dev